### ISSUE TITLE
Windows 7, MultiBit 0.5.16: createNewPeerGroup() fails => Multibit never gets out of "loading", can't open wallets

### ISSUE THREAD

**Issue by hkjn** [Multibit-Legacy/multibit#481]
Hey,
#### TL;DR

I think MultiBitService.java's [createNewPeerGroup()](https://github.com/jim618/multibit/blob/master/src/main/java/org/multibit/network/MultiBitService.java#L306) fails for my wallets, causing issues, but there is no logging to aid in digging further.
### Symptoms

I'm seeing an issue I can't find prior reports for. I created three wallets in Multibit some time ago and sent some Bitcoins to them (confirmed to have gone through via blockchain.info). I've never been able to successfully open these wallets since then, saw the org.jboss.netty.channel.ChannelException exception (see below) in the UI, and Multibit always claims to be "Connecting..", while wallets are "Synchronizing.." or "Waiting..".
#### Startup stuff. Blockchain is old, but that may incidental as I think the client gives up once it reaches the NullPointerException below. I suspect that the peergroup isn't set up correctly, since [MultiBitService.java:L162](https://github.com/jim618/multibit/blob/master/src/main/java/org/multibit/network/MultiBitService.java#L162) is reached, but not L164 (just after creating the peergroup):

20:55:03.475 [main] DEBUG org.multibit.network.MultiBitService - Blockstore is 'com.google.bitcoin.store.SPVBlockStore@1d2f0cb' 
20:55:03.476 [main] DEBUG org.multibit.network.MultiBitService - Creating blockchain ... 
20:55:03.479 [main] INFO  c.g.bitcoin.core.AbstractBlockChain - chain head is at height 264096:
v2 block: 
   previous block: <..>   time: [1381925788] Wed Oct 16 05:16:28 PDT 2013
20:55:03.479 [main] DEBUG org.multibit.network.MultiBitService - Created blockchain 'com.google.bitcoin.core.MultiBitBlockChain@18d708f' with height 264096 
20:55:03.479 [main] DEBUG org.multibit.network.MultiBitService - Creating peergroup ... 
20:55:03.728 [Timer-4] DEBUG o.m.model.exchange.ConnectHttps - checkServerTrusted authType = RSA 
20:55:03.940 [Timer-4] DEBUG o.multibit.exchange.TickerTimerTask - Got ticker for BTC USD 
20:55:03.940 [Timer-4] DEBUG o.multibit.exchange.TickerTimerTask - Exchange = MtGox 
20:55:13.272 [Timer-3] DEBUG o.m.v.swing.FileChangeTimerTask - Start of FileChangeTimerTask - run - enable = true 
20:55:13.272 [Timer-3] DEBUG o.m.v.swing.FileChangeTimerTask - End of FileChangeTimerTask - run 
20:55:17.258 [AWT-EventQueue-0] DEBUG o.m.controller.core.CoreController - Displaying view 'TRANSACTIONS_VIEW' 
20:55:17.258 [AWT-EventQueue-0] DEBUG o.m.viewsystem.swing.MultiBitFrame - Displaying view 'TRANSACTIONS_VIEW' 
20:55:17.259 [AWT-EventQueue-0] DEBUG o.m.v.swing.MultiBitTabbedPane - Set selected index = 2 
20:55:24.537 [main] ERROR org.multibit.network.MultiBitService - Error creating MultiBitService org.jboss.netty.channel.ChannelException Failed to create a selector. 
#### The following seems to be logged from [MultiBitService.java:L497](https://github.com/jim618/multibit/blob/master/src/main/java/org/multibit/network/MultiBitService.java#L497) and likely is just fallout of the peerGroup being null:

20:55:24.642 [main] ERROR org.multibit.network.MultiBitService - Could not add wallet 'C:\Users\z\AppData\Roaming\MultiBit\multibit.wallet' to the peerGroup as the peerGroup is null. This is bad.  
#### More peerGroup-related fallout..

20:55:24.664 [main] DEBUG org.multibit.MultiBit - Downloading blockchain 
java.lang.NullPointerException
    at org.multibit.MultiBit.main(MultiBit.java:540)
    at org.multibit.MultiBitInExecutableJar.main(MultiBitInExecutableJar.java:98)
20:55:24.664 [main] ERROR org.multibit.MultiBit - An unexpected error caused MultiBit to quit. 
20:55:24.665 [main] ERROR org.multibit.MultiBit - The error was 'java.lang.NullPointerException null' 
java.lang.NullPointerException
    at org.multibit.MultiBit.main(MultiBit.java:540)
    at org.multibit.MultiBitInExecutableJar.main(MultiBitInExecutableJar.java:98)
20:55:24.665 [main] ERROR org.multibit.MultiBit - Please read http://multibit.org/help_troubleshooting.html for help on troubleshooting. 
Exception in thread "main" java.lang.NullPointerException
    at org.multibit.viewsystem.swing.action.ExitAction.actionPerformed(ExitAction.java:82)
    at org.multibit.MultiBit.main(MultiBit.java:626)
    at org.multibit.MultiBitInExecutableJar.main(MultiBitInExecutableJar.java:98)

20:55:51.531 [AWT-EventQueue-0] ERROR org.multibit.network.MultiBitService - Could not add wallet 'E:\m1.wallet' to the peerGroup as the peerGroup is null. This is bad. 
#### The following one is also weird - there is indeed a E:/m2.info on this system.

20:57:45.796 [AWT-EventQueue-0] DEBUG o.m.model.bitcoin.WalletInfoData - Could not load walletinfo file 'E:\m2.info' 
20:57:45.796 [AWT-EventQueue-0] DEBUG o.m.model.bitcoin.WalletInfoData - Cause : E:\m2.info (The system cannot find the file specified) 

20:59:10.269 [AWT-EventQueue-0] DEBUG org.multibit.file.FileHandler - Start of secureDelete 
20:59:10.310 [AWT-EventQueue-0] DEBUG org.multibit.file.FileHandler - Result of delete of file 'E:\m2.wallet' was true 
20:59:10.310 [AWT-EventQueue-0] DEBUG org.multibit.file.FileHandler - End of secureDelete 
20:59:10.310 [AWT-EventQueue-0] DEBUG org.multibit.file.FileHandler - Saving wallet file 'E:\m2.wallet' ... 
20:59:10.310 [AWT-EventQueue-0] DEBUG org.multibit.file.FileHandler - ... done saving wallet file. 
20:59:10.310 [AWT-EventQueue-0] DEBUG org.multibit.file.FileHandler - Start of secureDelete 
20:59:10.310 [AWT-EventQueue-0] DEBUG org.multibit.file.FileHandler - End of secureDelete 
20:59:10.311 [AWT-EventQueue-0] DEBUG org.multibit.file.FileHandler - rememberFileSizesAndLastModified: Wallet filename E:\m2.wallet , walletFileSize 196 ,walletFileLastModified 1386219550310 ,walletInfoFileSize 273 ,walletInfoFileLastModified 1386219550311 
20:59:10.601 [AWT-EventQueue-0] DEBUG org.multibit.file.FileHandler - Saving wallet file 'E:\m2-data\wallet-backup\m2-20131204205910.wallet' ... 
20:59:10.602 [AWT-EventQueue-0] DEBUG org.multibit.file.FileHandler - ... done saving wallet file. 
20:59:10.602 [AWT-EventQueue-0] INFO  org.multibit.file.BackupManager - Written backup wallet files to 'E:\m2-data\wallet-backup\m2-20131204205910.wallet', 'E:\m2-data\wallet-backup\m2-20131204205910.info' 

21:01:13.275 [Timer-3] DEBUG o.m.v.swing.FileChangeTimerTask - Saving dirty wallet 'E:\m1.wallet'... 

21:01:13.276 [Timer-3] DEBUG org.multibit.file.FileHandler - Start of secureDelete 
21:01:13.317 [Timer-3] DEBUG org.multibit.file.FileHandler - Result of delete of file 'E:\m1.wallet' was true 
21:01:13.317 [Timer-3] DEBUG org.multibit.file.FileHandler - End of secureDelete 
21:01:13.317 [Timer-3] DEBUG org.multibit.file.FileHandler - Saving wallet file 'E:\m1.wallet' ... 
21:01:13.318 [Timer-3] DEBUG org.multibit.file.FileHandler - ... done saving wallet file. 

Thanks in advance for any help in rescuing these wallets / patching the issue.
